docs/pages/formset-customization.rst

```diff
@@ -159,3 +159,52 @@ Then use the appropriate names to render them in the html template:
     ...
     {{ Items }}
     ...
+
+Success messages
+----------------
+When using Django's `django.contrib.messages` framework, mixins are available in
+order to send success messages in a similar way to
+`django.contrib.messages.views.SuccessMessageMixin`. Ensure that
+`'django.contrib.messages.middleware.MessageMiddleware'` is included in the
+`MIDDLEWARE` section of `settings.py`.
+
+`extra_views.SuccessMessageMixin` is for use with views with multiple
+inline formsets. It is used in an identical manner to the Django mixin:
+
+.. code-block:: python
+
+    from extra_views import CreateWithInlinesView, SuccessMessageMixin
+    ...
+
+    class CreateOrderView(SuccessMessageMixin, CreateWithInlinesView):
+        model = Order
+        inlines = [ItemInline, ContactInline]
+        success_message = 'Order successfully created!'
```

Comment:
The only suggestion I have is to use an example with string formatting placeholders here, like in the Django documentation, and maybe even link to the Django documentation.

Reply:
Agreed, I've added all of those in!